### DISCUSSION TITLE
Keep audio device alive

### DISCUSSION THREAD

**Discussion by Uk0kA** [MusicPlayerDaemon/MPD#1983] [Ideas]
It would be great if there was some way to keep connected audio devices alive by sending a low level noise (silence) at a defined interval.

**Comment by Uk0kA** [comment]
Or perhaps send a silence signal prior to playback

**Comment by Badsheep** [comment]
Hi, I'd be interested too by this idea...

**Comment by Badsheep** [comment]
Is it not already possible ? See the option "always_on" [here](https://mpd.readthedocs.io/en/latest/user.html#configuring-audio-outputs) ?
